view/popup.html

```diff
@@ -93,16 +93,21 @@
         <div id="infoContent">
             <!-- ABOUT -->
             <div v-show="info === 'about'">
-                <p><strong>Authenticator</strong> for Google&trade; Authenticator,<br/>&copy; 2018 <a target='_blank' href='https://github.com/Authenticator-Extension'>Authenticator Extension</a>. Released under the MIT License.</p>
-                <p>Google&trade; is Google's trademark, Google Authenticator is Google’s trade name, those rights of ownership belong with Google Inc.</p>
-                <p><a target='_blank' href='https://github.com/LazarSoft/jsqrcode'>jsqrcode</a> Copyright jsqrcode authors. Licensed under the Apache License.</p>
-                <p><a target='_blank' href='https://github.com/zxing/zxing'>ZXing</a> Copyright ZXing authors. Licensed under the Apache License.</p>
-                <p><a target='_blank' href='http://blog.tinisles.com/2011/10/google-authenticator-one-time-password-algorithm-in-javascript/'>totp.js</a> Copyright its author. Licensed under the MIT License.</p>
-                <p><a target='_blank' href='http://caligatio.github.com/jsSHA/'>jsSHA</a> Copyright jsSHA authors. Licensed under the BSD License.</p>
-                <p><a target='_blank' href='http://davidshimjs.github.io/qrcodejs/'>qrcode.js</a> Copyright qrcode.js author. Licensed under the MIT License.</p>
-                <p><a target='_blank' href='https://code.google.com/p/crypto-js/'>crypto-js</a> Copyright crypto-js author. Licensed under the BSD License.</p>
-                <p><a target='_blank' href='https://www.google.com/fonts/specimen/Droid+Sans+Mono'>Droid Sans Mono</a> Copyright Steve Matteson. Licensed under the Apache License.</p>
+                <p><strong>Authenticator</strong> &copy; 2018 <a target='_blank' href='https://github.com/Authenticator-Extension'>Authenticator Extension</a>. Released under the MIT License.</p>
+                <p>Google is a trademark of Google LLC.</p>
```

Comment:
Do we use word Google in our current code?

Reply:
Only on the sync clock with Google button

Reply:
Then I think we can remove this line. What's your option?

Reply:
This line needs to be kept, "Google" is a registered trademark. Lawyers are pretty aggressive when dealing with trademarks, especially since they can be lost if they aren't defended properly.

http://tsdr.uspto.gov/#caseNumber=87786172&caseType=SERIAL_NO&searchType=statusSearch

Reply:
Emmm, okay. Let's keep it.